cmd/operator/main.go

```diff
@@ -155,6 +155,10 @@ func Main() int {
 		return 1
 	}
 
+	// CMO runs many tasks in parallel and the default values for rate limiting are too low.
+	config.QPS = 50
+	config.Burst = 100
```

Comment:
Can we comment on how we came up with these numbers?

Reply:
I'm still experimenting :)

Reply:
Still seeing throttling logs with `QPS = 50` and `Burst = 100` though less frequently.